docker-compose.yml

```diff
@@ -13,17 +13,17 @@ mysql:
     - MYSQL_DATABASE
     - MYSQL_USER
     - MYSQL_PASSWORD
-  ports:
-    - 3306:3306
+  expose:
```

Comment:
Changing these from `ports` to `expose` makes the containers inaccessible unless the developer adds their own proxy container. If you're concerned about port conflicts on the host, we could define the host ports for each container using an environment variable:

```yaml
ports:
  - ${WORDPRESS_PORT}:80
```